e2e/fabricaCommands.test.ts

```diff
@@ -68,7 +68,7 @@ describe("validate", () => {
 
   it("should validate custom config", () => {
     // Given
-    const fabricaConfig = `${commands.relativeRoot}/samples/fabricaConfig-2orgs-2channels-2chaincodes-tls-raft.json`;
+    const fabricaConfig = `${commands.relativeRoot}/samples/fabrica-config-hlf1.4-1org-raft.json`;
```

Comment:
It might be my whim, but I would add basic information about chaincodes to config file names.  
We can have many chaincodes in many different variants, so it's useful information.